src/scss/core/pages/product/_product.scss

```diff
@@ -1,7 +1,10 @@
 .thumbnails__container {
-  margin-top: 1rem;
+  max-height: 38.125rem;
```

Comment:
I very much don't like thumbs on the left. It's hard to make the thumbs carousel responsive 😭
IMO we should use a component with thumbs built-in and on the top/bottom, similar to [THIS](https://fancyapps.com/playground/vu)

Reply:
I don't like it either :D